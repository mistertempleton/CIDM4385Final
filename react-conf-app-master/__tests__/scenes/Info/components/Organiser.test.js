// @flow
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';

import Organiser from '../../../../app/scenes/Info/components/Organiser';

describe('Info - Organiser', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Organiser
          avatar="avatar"
          name="Tes Ting"
          onPress={() => {}}
          summary="We are testing."
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
