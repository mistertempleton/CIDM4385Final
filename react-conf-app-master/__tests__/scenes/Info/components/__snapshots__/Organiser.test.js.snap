exports[`Info - Organiser renders correctly 1`] = `
<View
  accessibilityComponentType={undefined}
  accessibilityLabel={undefined}
  accessibilityTraits={undefined}
  accessible={true}
  hasTVPreferredFocus={undefined}
  hitSlop={undefined}
  isTVSelectable={true}
  onLayout={undefined}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
      },
      Object {
        "backgroundColor": "white",
      },
    ]
  }
  testID={undefined}
  tvParallaxProperties={undefined}>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderBottomColor": "#CCC",
        "borderBottomWidth": 0.5,
        "flexDirection": "row",
        "flexGrow": 1,
        "flexShrink": 1,
        "padding": 17,
      }
    }>
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#F4F4F4",
            "borderRadius": 44,
            "height": 44,
            "overflow": "hidden",
            "width": 44,
          },
          undefined,
        ]
      }>
      <Image
        source={
          Object {
            "uri": "avatar",
          }
        }
        style={
          Object {
            "borderRadius": 0,
            "height": 44,
            "width": 44,
          }
        } />
    </View>
    <View
      style={
        Object {
          "flexGrow": 1,
          "flexShrink": 1,
          "paddingLeft": 17,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#383838",
            "fontSize": 17,
          }
        }>
        Tes Ting
      </Text>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "#666",
            "fontSize": 14,
            "fontWeight": "300",
            "marginBottom": 14,
          }
        }>
        We are testing.
      </Text>
    </View>
  </View>
</View>
`;
